File: simpleinterceptor-release/src/main/java/com/kangaroo/simpleinterceptor/SimpleInterceptor.java
package com.kangaroo.simpleinterceptor;

import android.content.Context;
import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.Request;
import okhttp3.Response;

/**
 * OkHttp拦截器
 */
public final class SimpleInterceptor implements Interceptor {

    public SimpleInterceptor(Context context) {
    }


    @Override public Response intercept(Chain chain) throws IOException {
        Request request = chain.request();
        return chain.proceed(request);
    }
}

File: simpleinterceptor-release/src/main/java/com/kangaroo/simpleinterceptor/SimpleInterceptor.kt
package com.kangaroo.simpleinterceptor

import android.content.Context
import okhttp3.Interceptor
import okhttp3.Response
import java.io.IOException

/**
 * OkHttp拦截器
 */
class SimpleInterceptor(context: Context?) : Interceptor {
    @Throws(IOException::class)
    override fun intercept(chain: Interceptor.Chain): Response {
        val request = chain.request()
        return chain.proceed(request)
    }
}